backend-mock/apps.js

```diff
@@ -1,5 +1,23 @@
+const express = require("express");
+const router = express.Router();
 const util = require("./util");
 
+router.post("/install/:id", (req, res) => {
+  console.info("call to /api/v1/apps/install for app", req.params.id);
+  // send information that btc-pay is currently installing
+  util.sendSSE("install", { id: "mempool" });
+  setTimeout(() => {
+    installApp();
+  }, 5000);
+  res.status(200).send();
+});
+
+router.post("/uninstall/:id", (req, res) => {
+  console.info("call to /api/v1/apps/uninstall for app", req.params.id);
+  // TODO
```

Comment:
What's the todo here?

Reply:
Doing some actions like the installing does. Not really important atm :)

Reply:
Wanna comment that?

Reply:
Sure, done! :+1: